### DISCUSSION TITLE
Packet is not arriving when multicasting

### DISCUSSION THREAD

**Discussion by DoyunShin** [ANLAB-KAIST/KENSv3#249] [Project Q&A 3. Internet Layer (IP/Routing)] [answered]
![image](https://github.com/ANLAB-KAIST/KENSv3/assets/60532811/7545d50e-0b81-4977-bd3f-47e053f76634)

I'm now sending a packet with
00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
00 00 00 00 00 00 00 00 00 00 **0a 00 01 01 e0 00 
00 05** 02 01 00 42 01 01 00 0a 01 00 00 00 00 00 
00 00 00 00 00 00 00 00 00 00 ff ff ff 00 00 3c 
00 00 

But, Nothing appears on pcap.

**Comment by Pusnow** [comment, marked as answer]
Did you fix this problem? If it persists, please send your code to TA. cs341.pj@kaist.ac.kr

**Reply by DoyunShin** [reply]
Oh, Now I found what was the error. I just added the timer without thinking about nano second.
